source/features/clone-branch.tsx

```diff
@@ -62,20 +62,20 @@ async function cloneBranch(event: delegate.Event<MouseEvent, HTMLButtonElement>)
 	}
 
 	textFieldEdit.set(
-		select<HTMLInputElement>('.js-branch-search-field')!,
+		select<HTMLInputElement>('[aria-label="Search branches…"]')!,
```

Comment:
Let’s use its public `name`, that shouldn't ever change
```suggestion
		select<HTMLInputElement>('[name="query"]')!,
```